Add some missing emoji variants

eranl opened the pull request:
The logic that filters out multi-skin-tone variants was too aggressive, and dropped some variants that have the same skin tone twice in their codes.

BTW, as I commented in the code, it's possible to add some multi-skin-tone variants too, and it looks pretty good in the app IMO.

Helium314 commented:
> BTW, as I commented in the code, it's possible to add some multi-skin-tone variants too

Would there be any reason not to use them? The comment about the Openboard implementation might be about an older emoji keyboard version without popups?

eranl commented:
> > BTW, as I commented in the code, it's possible to add some multi-skin-tone variants too
> 
> Would there be any reason not to use them? The comment about the Openboard implementation might be about an older emoji keyboard version without popups?

Added them, and updated comment.

BTW, I've noticed that some of the single-skin-tone variants are not displayed correctly in WhatsApp. They seem to have their own way of dealing with variants. Once sent, it shows up fine on WhatsApp Web. I checked a few other apps, and it works fine there.

Helium314 commented:
Maybe they use their own font? That could explain why it shows up fine on WhatsApp Web.
